2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -103,6 +119,7 @@ collector {
     }
 
     # Incoming events are stored in a buffer before being sent to Kinesis/Kafka.
+    # Note: Buffering is not supported by NSQ; will be ignored    
```

Comment:
I don't think it really applies to anything since the buffer settings are producer-related and not consumer-related so they shouldn't be in stream-enrich either.